app/src/main/java/com/khmelenko/lab/travisclient/adapter/PullRequestsListAdapter.java

```diff
@@ -76,7 +76,11 @@ private void bindPullRequest(BranchViewHolder holder, int position, RequestData
             }
         }
 
-        holder.mBuildView.setPullRequestData(request, relatedBuild, relatedCommit);
+        holder.mBuildView.setPullRequestTitle(request);
+        holder.mBuildView.setCommit(relatedCommit);
+        holder.mBuildView.setState(relatedBuild);
+        holder.mBuildView.setTitle(
+                holder.mBuildView.getContext().getString(R.string.pull_request_number, request.getPullRequestNumber()));
     }
```

Comment:
@DreierF shouldn't this class have `BuildViewHolder` as ViewHolder? then we can remove `BranchViewHolder` and `item_pull_request.xml`